.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "misc/cache-digest.js"]
 	path = misc/cache-digest.js
 	url = https://github.com/h2o/cache-digest.js.git
+[submodule "misc/mruby-mtest"]
+	path = misc/mruby-mtest
+	url = https://github.com/iij/mruby-mtest.git
```

Comment:
Please use `misc/dump-github-repository.pl` to dump mruby-mtest into `misc/mruby-mtest`. Run the perl script without arguments to find out how to use the program.

The reason we should do so is because tarballs created by GitHub will not include submodules; files required for building and testing H2O should never be a git-submodule.